python/examples/atomistic/3-atomistic-model-with-nl.py

```diff
@@ -163,11 +163,12 @@
 # %%
 #
 # The data and metadata inside the ``neighbors`` object do not contain information about
-# the ``cutoff`` and whether this is a full or half neighbor list. To account for this,
-# metatensor neighbor lists are always stored together with
+# the ``cutoff``, whether this is a full or half neighbor list, and whether it is
+# restricted to distances strictly below the cutoff (``strict=True``). To account
+# for this, metatensor neighbor lists are always stored together with
```

Comment:
We should have a small explanation about when someone would want to use non-strict list & how to properly hand them.